bigquery_etl/cli/query.py

```diff
@@ -1636,7 +1634,10 @@ def deploy(
 
 def _attach_metadata(query_file_path: Path, table: bigquery.Table) -> None:
     """Add metadata from query file's metadata.yaml to table object."""
-    metadata = Metadata.of_query_file(query_file_path)
+    try:
+        metadata = Metadata.of_query_file(query_file_path)
+    except FileNotFoundError:
+        return
```

Comment:
some older queries might not have `metadata.yaml` files

Reply:
Should we add them instead of making `metadata.yaml` optional for future tables? I'm just thinking we should encourage people to add descriptions and context around the tables they create and what purpose it serves.

Reply:
Or maybe there should be a place where explicitly we have to allow no `metadata.yaml` for a specific table? Like we do with skip-dry-run. So it's more "intentional".

Reply:
We should probably add metadata.yaml files instead of investing time in curating another skip list.

Reply:
I opened https://github.com/mozilla/bigquery-etl/issues/3416 to follow up on this

Reply:
thanks!